docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -47,6 +57,22 @@ If there is an error please [contact support](mailto:pulsesupport@codacy.com).
 -   The set of changes in a deployment is the list of commits in the corresponding pull request.
 -   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
 
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The following is a detailed description of how the Pulse Bitbucket integration automatically detects incidents:
+
+### Use pull request reverts (based on default branch)
+
+-   Pulse considers an incident a reverted pull request that **targets the default branch** of the repository when the revert operation was executed through the Bitbucket UI originating a branch which name starts with `revert-`.
+-   The incident creation date is the timestamp when the reverted pull request was initially merged.
+-   Pulse associates all the Bitbucket user groups of the author of a revert pull request with the corresponding incident, excluding the user groups with less than two members.
```

Comment:
if you see the definition of [Incidents](https://docs.pulse.codacy.com/cli/cli/#3-pushing-incidents), you'll find that unlike for Deployments, we don't require specifying a team

the team will be assumed from whatever deployment a given incident ends up being associate with

Reply:
not and I'm sure if it's worth specifying, but the `system` is inferred from the repository name

Reply:
Thanks! 👍  I updated the third bullet with the system information.